tests/test_density.py

```diff
@@ -0,0 +1,170 @@
+from argparse import ArgumentParser
+import numpy as np
+import math
+from parcels import Field, Grid, JITParticle, ScipyParticle, Variable
+
+
+def GradientClimber(particle, grid, time, dt):
+    f_lat = dt / 1000. / 1.852 / 60.
+    f_lon = f_lat / math.cos(particle.lat*math.pi/180)
```

Comment:
Can't we use the `mesh=flat` option here, to have field.py do these conversions automatically? In that way, users can use the same GradientClimber on both spherical and flat meshes